pym/portage/dbapi/bintree.py

```diff
@@ -568,7 +579,7 @@ def _populate(self, getbinpkgs=False, getbinpkg_refresh=True):
 				metadata[_instance_key(cpv)] = d
 				path = d.get("PATH")
 				if not path:
-					path = cpv + ".tbz2"
+					path = cpv + "." + self.compression["suffix"]
```

Comment:
For code like this we should also support "tbz2" for backward compatibility. Using xpak for all compression types is good, since compression_probe allows us to quickly determine the compression whenever we need to.